notebooks/analysis/MCMICRO_with_IDC.ipynb

```diff
@@ -0,0 +1,3664 @@
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/7806a77a-825a-4bc5-a2f1-561adfc3c996.dcm MCMICRO-example-IDC/DICOM/7806a77a-825a-4bc5-a2f1-561adfc3c996.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/c4695fd6-6511-4d40-8338-f3a35a20675c.dcm MCMICRO-example-IDC/DICOM/c4695fd6-6511-4d40-8338-f3a35a20675c.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/72bfba0c-6086-439a-959c-0f596e777f43.dcm MCMICRO-example-IDC/DICOM/72bfba0c-6086-439a-959c-0f596e777f43.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/6e7afd1b-a009-461b-9f56-df9686e8188e.dcm MCMICRO-example-IDC/DICOM/6e7afd1b-a009-461b-9f56-df9686e8188e.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/dc6b3aff-0421-47c0-9648-bbf95f51a95d.dcm MCMICRO-example-IDC/DICOM/dc6b3aff-0421-47c0-9648-bbf95f51a95d.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/50b64963-995f-4113-8f61-50425f06052b.dcm MCMICRO-example-IDC/DICOM/50b64963-995f-4113-8f61-50425f06052b.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/8c193e38-ea92-4350-9348-6efd18db16b8.dcm MCMICRO-example-IDC/DICOM/8c193e38-ea92-4350-9348-6efd18db16b8.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/921cb689-0180-4ee6-a298-98c403cf2c62.dcm MCMICRO-example-IDC/DICOM/921cb689-0180-4ee6-a298-98c403cf2c62.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/0548fd40-cc50-47ca-9ed6-879706ea2a20.dcm MCMICRO-example-IDC/DICOM/0548fd40-cc50-47ca-9ed6-879706ea2a20.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/819cbef0-12da-4e81-a55d-645f11dab3f0.dcm MCMICRO-example-IDC/DICOM/819cbef0-12da-4e81-a55d-645f11dab3f0.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/b72a54fe-3091-4848-8661-f5fa3b0f0c7d.dcm MCMICRO-example-IDC/DICOM/b72a54fe-3091-4848-8661-f5fa3b0f0c7d.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/8c54f6e4-7226-4a55-b63b-4d4b74622ca5.dcm MCMICRO-example-IDC/DICOM/8c54f6e4-7226-4a55-b63b-4d4b74622ca5.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/1e6a68ea-a45a-4d54-aa3a-090a8b0b772e.dcm MCMICRO-example-IDC/DICOM/1e6a68ea-a45a-4d54-aa3a-090a8b0b772e.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/d490a51b-5f24-4f5a-bf7b-8b38c33200ae.dcm MCMICRO-example-IDC/DICOM/d490a51b-5f24-4f5a-bf7b-8b38c33200ae.dcm\n",
+      "cp s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/8db6a451-993d-41ea-804c-76705d155e03.dcm MCMICRO-example-IDC/DICOM/8db6a451-993d-41ea-804c-76705d155e03.dcm\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Download sample fluorescence data from IDC\n",
+    "\n",
+    "!s5cmd --no-sign-request --endpoint-url https://s3.amazonaws.com cp \"s3://idc-open-data/fc078201-b5c5-47d1-92da-3c81dd8cc2c5/*\" MCMICRO-example-IDC/DICOM/"
```

Comment:
Instead of using `s5cmd`, it is more convenient to `pip install --upgrade idc-index` and then do this:

```!idc download-from-selection --download-dir=/media/volume/sdb/MCMICRO-example-IDC/DICOM/ --dir-template=""   --series-instance-uid=1.3.6.1.4.1.5962.99.1.2343322182.1764456793.1655905763910.4.0`

Among other things, the command above will check if the destination directory has enough space, and will report download progress, which is quite helpful for this large image series.

It would be best to parameterize this and other cell that work with input or output images with the destination directory. I tested this on a cloud VM that did not have enough disk space (the image is ~60GB), and I had to use a mounted volume. On the first try, I did not realize that the subsequent cells depend on the files in the repo (parameters/markers), and when I first tried to run the analysis on the data downloaded and converted in a directory in that volume, it failed. It worked after I copied the missing files.